.github/workflows/github-release.yml

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details
+
+name: Release
```

Just to avoid confusion with Pypi, what about mentioning `GitHub`?
```suggestion
name: GitHub Release
```